examples/bevy_ggrs/src/main.rs

```diff
@@ -1,7 +1,8 @@
-use bevy::{log::LogPlugin, prelude::*, tasks::IoTaskPool};
+use bevy::{log::LogPlugin, prelude::*};
 use bevy_ggrs::{GGRSPlugin, Session};
+use bevy_matchbox::{MatchboxSocket, OpenSocketExt};
 use ggrs::SessionBuilder;
-use matchbox_socket::{PeerState, SingleChannel, WebRtcSocket};
+use matchbox_socket::{ChannelConfig, PeerState, SingleChannel, WebRtcSocketBuilder};
```

Comment:
Perhaps better to encourage use of the re-export?

Reply:
Have used matchbox_socket::prelude::*;